src/main/resources/db/changelog/db.changelog-RDM-8747.xml

```diff
@@ -0,0 +1,15 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-8747" author="tobias.derose@hmcts.net">
+        <addColumn tableName="case_data">
+            <column name="supplementary_data" type="jsonb">
+                <constraints nullable="true"/>
+            </column>
+            <column name="supplementary_data_last_modified" type="timestamp">
+                <constraints />
```

Comment:
we have this as initially null?  and then will be gradually populated whenever supplementary data is written? Would this cause problems to the LS sql query? let's verify

Reply:
Yes it has been tested .  When cases are created  supplementary_data_last_modified is null. LogStash keep running and execution queries without error.

Reply:
ok cool, having it as initially null is fine then.